worlds/sc2/Items.py

```diff
@@ -27,39 +28,183 @@ def get_full_item_list():
+                                  Can switch into Defender Mode to provide siege support.
+                                  """),
+    ItemNames.VALKYRIE: ItemData(19 + SC2WOL_ITEM_ID_OFFSET, "Unit", 19, SC2Race.TERRAN,
+                                 classification=ItemClassification.progression, origin={"bw"},
+                                 description="""
+                                 Advanced anti-aircraft fighter. 
+                                 Able to use cluster missiles that deal area damage to air targets.
+                                 """),
+    ItemNames.WIDOW_MINE: ItemData(20 + SC2WOL_ITEM_ID_OFFSET, "Unit", 20, SC2Race.TERRAN,
+                                   classification=ItemClassification.progression, origin={"ext"},
+                                   description="""
+                                   Robotic mine. Launches missiles at nearby enemy units while burrowed. 
+                                   Attacks deal splash damage in a small area around the target. 
+                                   Widow Mine is revealed when Sentinel Missile is on cooldown.
+                                   """),
+    ItemNames.CYCLONE: ItemData(21 + SC2WOL_ITEM_ID_OFFSET, "Unit", 21, SC2Race.TERRAN,
+                                classification=ItemClassification.progression, origin={"ext"},
+                                description="""
+                                Mobile assault vehicle. Can use Lock On to quickly fire while moving.
+                                """),
 
     # Some other items are moved to Upgrade group because of the way how the bot message is parsed
-    ItemNames.PROGRESSIVE_TERRAN_INFANTRY_WEAPON: ItemData(100 + SC2WOL_ITEM_ID_OFFSET, "Upgrade", 0, SC2Race.TERRAN, quantity=3),
-    ItemNames.PROGRESSIVE_TERRAN_INFANTRY_ARMOR: ItemData(102 + SC2WOL_ITEM_ID_OFFSET, "Upgrade", 2, SC2Race.TERRAN, quantity=3),
-    ItemNames.PROGRESSIVE_TERRAN_VEHICLE_WEAPON: ItemData(103 + SC2WOL_ITEM_ID_OFFSET, "Upgrade", 4, SC2Race.TERRAN, quantity=3),
-    ItemNames.PROGRESSIVE_TERRAN_VEHICLE_ARMOR: ItemData(104 + SC2WOL_ITEM_ID_OFFSET, "Upgrade", 6, SC2Race.TERRAN, quantity=3),
-    ItemNames.PROGRESSIVE_TERRAN_SHIP_WEAPON: ItemData(105 + SC2WOL_ITEM_ID_OFFSET, "Upgrade", 8, SC2Race.TERRAN, quantity=3),
-    ItemNames.PROGRESSIVE_TERRAN_SHIP_ARMOR: ItemData(106 + SC2WOL_ITEM_ID_OFFSET, "Upgrade", 10, SC2Race.TERRAN, quantity=3),
+    ItemNames.PROGRESSIVE_TERRAN_INFANTRY_WEAPON: ItemData(100 + SC2WOL_ITEM_ID_OFFSET, "Upgrade", 0, SC2Race.TERRAN,
+                                                           quantity=3,
+                                                           description=f"""
+                                                           Increases damage of Terran infantry units. 
+                                                           ${WEAPON_ARMOR_UPGRADE_NOTE}
+                                                           """),
```

Comment:
This doesn't seem like a good usage of multi-line strings. What this will expand to is something like this:
> \n
> [58 spaces]Increases damage of Terran infantry units.\n
> [58 spaces]\n
> [4 spaces]$Must be researched during the mission if the mission type isn't set to auto-unlock generic upgrades.\n
> \n
> [58 spaces]\n

* The spacing is all over the place, and large
* The `$` is visible before the substitution
* There are many extraneous newlines

Recommended change: wrap this in parentheses and open / close strings.
```python
    ItemNames.PROGRESSIVE_TERRAN_INFANTRY_WEAPON: ItemData(100 + SC2WOL_ITEM_ID_OFFSET, "Upgrade", 0, SC2Race.TERRAN,
                                                           quantity=3,
                                                           description=(
                                                           "Increases damage of Terran infantry units. "
                                                           f"{WEAPON_ARMOR_UPGRADE_NOTE}"
                                                           )),
```

Reply:
Used `inspect.cleandoc` on multi-line string and reformatted the file